cli/js/lib.deno.ns.d.ts

```diff
@@ -10,6 +10,31 @@ declare namespace Deno {
   /** Reflects the NO_COLOR environment variable: https://no-color.org/ */
   export let noColor: boolean;
 
+  export type TestFunction = () => void | Promise<void>;
+
+  export interface TestDefinition {
+    fn: TestFunction;
+    name: string;
+  }
+
+  export function test(t: TestDefinition): void;
+  export function test(fn: TestFunction): void;
+  export function test(name: string, fn: TestFunction): void;
+
+  export interface RunTestsOptions {
+    exitOnFail?: boolean;
+    only?: RegExp;
+    skip?: RegExp;
+    disableLog?: boolean;
+  }
+
+  export function runTests(opts: RunTestsOptions): Promise<void>;
+
+  export function runIfMain(
+    meta: ImportMeta,
+    opts?: RunTestsOptions
+  ): Promise<void>;
```

Comment:
Remove its usages in `cli/js/net_test.ts` and `cli/js/tls_test.ts`, replace its usage in `cli/js/unit_tests.ts` with an explicit `import.meta.main` check. Then you can get rid of this.